lib/chef/resource/windows_certificate.rb

```diff
@@ -92,16 +90,22 @@ class WindowsCertificate < Chef::Resource
         default: false,
         introduced: "16.8"
 
-      action :create, description: "Creates or updates a certificate" do
-        # Extension of the certificate
-        ext = ::File.extname(new_resource.source)
+      property :output_path, String,
+        description: "A path on the node where a certificate object (pfx, pem, cer, key, etc) can be exported to.",
+        introduced: "16.10"
+
+      action :create do
+        description "Creates or updates a certificate."
+
+        ext = get_file_extension(new_resource.source)
 
         # PFX certificates contains private keys and we import them with some other approach
         import_certificates(fetch_cert_object(ext), (ext == ".pfx"))
       end
 
       # acl_add is a modify-if-exists operation : not idempotent
-      action :acl_add, description: "Adds read-only entries to a certificate's private key ACL" do
+      action :acl_add do
+        description "Adds read-only entries to a certificate's private key ACL."
```

Comment:
This needs to be done the previous way. This way where description is its own method doesn't work